app/assets/stylesheets/signin_form.scss

```diff
@@ -0,0 +1,55 @@
+@import "bootstrap";
+
+body {
+  background-color: #fff;
+  padding-top: 30px;
+}
+
+.border-form-div {
```

Comment:
I don't feel quite comfortable with this class name, it contain html element names on the class name...and also a property, so I don't know what it is suppose to do...